test.py

```diff
@@ -90,26 +119,45 @@
     runGaugeB               = False 
     runGaugeC               = False 
     runGaugeD               = False 
-    runGaugeDelta           = False 
-    default_der = args.deriv
+    runGaugeDelta           = False
+    print('Jobs chosen') 
+    
     ################### Define V potential ######################
     if(computeNuclearPotential):
-        if args.potential == 'point_charge':
-            Peps = vp.ScalingProjector(mra,prec/10)
-            f = lambda x: nucpot.point_charge(x, origin, Z)
-            V_tree = Peps(f)
-        elif args.potential == 'coulomb_HFYGB':
-            Peps = vp.ScalingProjector(mra,prec/10)
-            f = lambda x: nucpot.coulomb_HFYGB(x, origin, Z, prec)
-            V_tree = Peps(f)
-        elif args.potential == 'homogeneus_charge_sphere':
-            Peps = vp.ScalingProjector(mra,prec/10)
-            f = lambda x: nucpot.homogeneus_charge_sphere(x, origin, Z, atom)
-            V_tree = Peps(f)
-        elif args.potential == 'gaussian':
-            Peps = vp.ScalingProjector(mra,prec/10)
-            f = lambda x: nucpot.gaussian(x, origin, Z, atom)
-            V_tree = Peps(f)
+        V_tot = vp.FunctionTree(mra)
+        V_tot.setZero()
+        for atom, origin in coordinates.items():
```

Comment:
All this stuff should go in the nuclear potential file. Here you just want to (1) parse the gepmetry (2) send it to the appropriate function which will return the nuclear potential